server/chat-commands/info.ts

```diff
@@ -2198,12 +2288,16 @@ const commands = {
 
 		Config.potd = target;
```

Comment:
```suggestion
		return this.errorReply(`Sorry, PotD is currently unsupported.`);

		Config.potd = target;
```

Reply:
I hope there's no lint rule against this. We might need to comment out _everything_ after the error.

Reply:
A lint rule against.... putting inaccessible code in? I dont think this has never NOT been a lint rule

Reply:
I commented out the rest of the code and added the early return.